CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Utility that generate an Astarte Device ID
```

Comment:
I would write something like: Add methods that generate an Astarte Device ID using either a random UUIDv4 or by deriving an UUIDv5 using unique data.